articles/flows/concepts/device-auth.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Device Authorization Flow
+description: Learn how the Device Authorization flow works and why you should use it for input-constrained devices, such as smart TVs and media consoles. For use with native apps.
+topics:
+  - input-constrained-devices
+  - client-credentials
+  - api-authorization
+  - grants
+  - flows
+  - authorization
+  - mobile-apps
+  - desktop-apps
+  - native-apps
+contentType: concept
+useCase:
+  - add-login
+  - secure-api
+  - call-api
+---
+# Device Authorization Flow
+
+With devices that connect wirelessly to the internet, rather than authenticate the user directly, the device asks the user to go to a link on their computer or smartphone and authorize the device. This avoids a poor user experience for devices that do not have an easy way to enter text. To do this, device apps use the Device Authorization Flow (drafted in [OAuth 2.0](https://tools.ietf.org/html/draft-ietf-oauth-device-flow-15)), in which they pass along their Client ID to initiate the authorization process and get a token.
+
+## How it works
+
+The Device Authorization Flow contains two different paths; one occurs on the device requesting authorization and the other occurs in a browser. The browser flow path, wherein a device code is bound to the session in the browser, occurs in parallel to part of the device flow path.
+
+![Device Authorization Sequence](/media/articles/flows/concepts/auth-sequence-device-auth.png)
```

Comment:
Thanks for pointing out the problem with the endpoint names. This was the language we used on all the other flow docs (/token) and in some other areas of the site, but I see that it is inaccurate. I'll fix this here and open a separate PR to take care of the other site areas. Great feedback!

Reply:
@ziluvatar, we saw step 3 of the browser flow (the "consent" piece) as being the authorization, which is how it is represented in our other flows. That said, we did struggle with how to show the two flows visually.

Reply:
I have a request in to graphics to update the image with the proper endpoint names.

Reply:
> we saw step 3 of the browser flow (the "consent" piece) as being the authorization, which is how it is represented in our other flows. 

Ok... consent is optional (and authenticate too in case of using SSO, but that's another story), in any case in step 4, for me, it is the authorization server (Auth0 Tenant) the one that does something, and it does not generate an access token, it activates or marks as authorized the device.
Maybe it is like that in other flows because Auth0 Tenant would return the tokens to the client in next step, however, with two flows here: either the generation goes to the device flow or we should rename the step for the browser flow.

Reply:
Makes sense. Ok, I've asked Stacy to change step 4 in the Browser Flow to "Marks device as authorized". Will leave this open until I get the graphic back and have added it to the PR.

Reply:
@ziluvatar , diagram has been updated.